### ISSUE TITLE
Issues in documentation

### ISSUE THREAD

**Issue by kad** [bigtreetech/BIGTREETECH-SKR-MINI-MZ#2]
Pinout file:
 * CNC_FAN1/CNC_FAN2, X_STOP, Y_STOP, Z_STOP, TB, T0 boxes are drawn such way that it is not matching connector orientation. Places of signals are not clear.
 * RGB - should it be marked as "neopixel" like on other boards?
 * SWD - not described.
 * Filename: would be good to name it in English (applicable to all files).

SKR mini MZ V1.0-Instruction Manual(1).pdf: 
 * Rename it without `(1)`
 * "MEGA Zero" -> should it be marked as "Anycubic Mega Zero" ?
 * On page 2: "Support BIGTREETECH-3.5 TFT、LCD12864" -> should it be "Supports BIGTREETECH TFT displays and original Anycubic LCD12864" ? Or if it is about the modes of BTT's TFT displays, maybe rephrase it more clearly.
 * Page 3, "12)" -> will be better to rephrase: "support connection of: ..." and then bulleted list of connectors.
 * Page 3, Motherboard parameters would be good to clearly say that RCT version of STM32 has only 256kb of flash.
 * Page 5: If you want to point to windows usb driver, point to exact location, not just to the list of repositories on github.
 * Page 5: in English manual, please use Windows English UI on screenshots.
 * Page 6: "motherboard wiring diagram" picture:
   * RGB in reality "Neopixel" (WS281x) port, right? it is not really generic RGB port.
   * "Neopixel" connector marking is in reality "PWR-Det" according to pinout file.
 * Page 7, pinout picture, see above comments for pinout file.
 * Page 7, "If you need to use the Filament Break Detection function," -> those jumpers are about "sensorless homing", right? not really about filament break detection.
 * Page 8: what is "BOOT" mode? what it is useful for? Description of it?
 * Page 8: firmware? point to right repository, not to the list of repositories.
 * Page 9: pointer to dropbox file: in this file, windows screenshots are not in English like the rest of document. not useful for many of people.
 * Page 9: "7)", note about heat bed: it would be good if it will be stated in the document what are the limits for heatbed supported by this board. (e.g. 12V, 10A max. / 24V - xxA max), so people can predict need of external MOSFET.

**Comment by pappicio** [comment]
no answer until now.... wow!
